documentation/Using-the-SDK/Advanced/Creating-Your-Own-DataSource.md

```diff
@@ -0,0 +1,204 @@
+# Abstract
+
+This document outlines a scenario where you want to implement a Data Source
```

Comment:
nit: for the entire file, I think "Data Source" should be either `DataSource` or "data source"